app/code/Magento/QuoteGraphQl/Model/CartMutationsAllowedInterface.php

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\QuoteGraphQl\Model;
+
+use Magento\Framework\GraphQl\Exception\GraphQlNoSuchEntityException;
+
+/**
+ * Service for checking that the shopping cart operations
+ * are allowed for current user
+ */
+interface CartMutationsAllowedInterface
```

Comment:
All classes/interfaces should be named using nouns. Please consider making changes across the review. In this specific case something like `Authorization\CartMutation::isAllowed` sounds better